src/main/groovy/com/calgaryscientific/gradle/VeracodeWorkflow.groovy

```diff
@@ -128,6 +149,18 @@ class VeracodeWorkflow {
 
         log.info("buildStatus: " + buildStatus)
 
+        // Previous Scan is complete
+        if (buildStatus == "Results Ready") {
+            log.info("Retrieving build list to obtain latest build_id")
+            Node buildList = XMLIO.writeXmlWithErrorCheck(VeracodeBuildList.getSandboxFile(outputDir, app_id, sandbox_id), veracodeAPI.getBuildListSandbox())
+            // Get the last build ID
+            build_id = VeracodeBuildList.getLatestBuildID(buildList)
+            log.info("Latest build_id: " + build_id)
+            newFlaws = printReportSummaryAndCheckForNewFlaws(veracodeAPI, outputDir, build_id)
```

Comment:
sounds like this function is doing 2 things .... we could break this to printReportSummary

and later down, we could try to use isNewFlaws?